### ISSUE TITLE
MogaSerial disables Internet connection

### ISSUE THREAD

**Issue by Alpstar72** [Zel-os/MogaSerial#8]
Hi,
I am a new MogaSerial user running MogaSerial V1.5.1 on an ASUS gaming laptop with Win7 professional 64bit. I have successfully connected my Moga Pro 2 bluetooth controller and started playing BioShock Infinite. However, whenever I connect the controller (using SCP/Xbox drivers), my internet (I use a Broadcom 802.11ac wireless adapter) shuts down. I have no internet connection, cannot surf the web or connect to STEAM, so I can only play offline. The same happens when I use the vJoy driver in MogaSerial. As soon as I disconnect the controller by pressing the button in MogaSerial, the internet comes back. This is frustrating since  although I can play a game, STEAM does not allow online game synchronisation (saving progress) without internet conncection. 
Is there any easy solution ?
Best regards !

**Comment by Zel-os** [comment]
Unfortunately, from everything I've been able to tell this is a common problem with integrated wireless & Bluetooth adapters, and the best I can do is refer you to the [Microsoft knowledgebase article about the issue](https://answers.microsoft.com/en-us/windows/forum/windows_10-networking/wifi-limiteddisconnected-when-bluetooth-is-enabled/8871aae8-d975-4906-88b7-a62ce3127df4).

Bluetooth adapters put out a lot of noise during the connection process, and if the channel it picks happens to overlap with the one wi-fi is using, they appear to cause a lot of interference with each other.  It seems to vary wildly between devices and driver versions as well.  The downside is that there's nothing I've found that can be done within the code of MogaSerial to compensate.

Duplicate of issue https://github.com/Zel-os/MogaSerial/issues/5
